src/components/Tabs.js

```diff
@@ -113,7 +114,12 @@ Tabs.propTypes = {
     handleSelect: PropTypes.func,
     selectedTab: PropTypes.string,
     activeLinkStyle: PropTypes.object,
-    visibleTabStyle: PropTypes.object
+    visibleTabStyle: PropTypes.object,
+    renderActiveTabContentOnly: PropTypes.bool
+};
+
+Tabs.defaultProps = {
```

Comment:
I don't think we need this here as accessing a prop that was not provided will result in `false`, effectively setting the default value to `false` for us. We might want to change that in the future if we decide to change the default though.